### DISCUSSION TITLE
Is this is main discussion forum around mobx? Any others?

### DISCUSSION THREAD

**Discussion by artemis-prime** [mobxjs/mobx#3060] [General]
Is this is main discussion forum around mobx? Any others?

**Comment by mweststrate** [comment]
Stack overflow or here indeed.

**Reply by artemis-prime** [reply]
Ok then, here's my question sir?  If one's goal is to slowly transition a Redux app to Mobx, is there any reason why one could not do it gradually and piecemeal?  For example, have both in the system at the same time, go page by page, or store domain by store domain.  I've always imagined it should be fine, but have never tried.  Thanks for all the outstanding work Michel!  Mobx rocks.

**Reply by artemis-prime** [reply]
I'll start a new thread!

**Comment by mweststrate** [comment]
yeah I'd do it reducer per reducer, and store the observable object that
holds the mobx state at that place in the redux tree instead, so that in
the state tree all the leaves become mobx observable objects piece by piece

**Reply by artemis-prime** [reply]
Can we take this up in the separate question I asked (for the benefit of future seekers)?
